package.json

```diff
@@ -43,6 +43,7 @@
   "dependencies": {
     "@aws-sdk/client-s3": "^3.438.0",
     "applicationinsights": "^2.9.0",
+    "govuk-frontend": "^4.7.0",
```

Comment:
Can bump to `4.8.0` now 🙂

Reply:
This is adding the new GOV.UK frontend module - but the (really) old ones are still there too. Is there a plan to remove those separately?

Reply:
> Can bump to `4.8.0` now 🙂

Updated to 4.8.0 but I think internal should't use crown logo as far as I know and current logo is not from this but from govuk-template-jinja so may be can update/remove in separate ticket.